pkg/gofr/http/middleware/config.go

```diff
@@ -9,6 +9,8 @@ import (
 	"gofr.dev/pkg/gofr/config"
 )
 
+const LogDisableProbeKey = "LOG_DISABLE_PROBES"
```

Comment:
The constant should have a clear comment about its purpose

Or simpl prefix it with EnvVar

```suggestion
const EnvVarLogDisableProbeKey = "LOG_DISABLE_PROBES"
```